setup.py

```diff
@@ -6,13 +8,33 @@ def readme():
         return f.read()
 
 
+def _is_old_setuptool():
+    """
+    Return True if setuptool is older than 20.5 which version support
+    environment markers for 'install_requires':
+        https://www.python.org/dev/peps/pep-0508/#environment-markers
+    """
+    major, minor = setuptools.__version__.split('.')[:2]
+    version = int(major) + int(minor) / 10.0
```

Comment:
You can compare tupels here, `(int(major), int(minor))  < (20, 5)` which is easier to understand IMHO.

Reply:
Changed to tuple compare. Thanks.